src/process_utility.c

```diff
@@ -2861,14 +2858,14 @@ process_altertable_start_view(ProcessUtilityArgs *args)
 
 	/* Check if this is an internal view of a continuous aggregate and give
 	 * error if attempts are made to alter them. */
-	namestrcpy(&view_name, get_rel_name(relid));
-	namestrcpy(&view_schema, get_namespace_name(get_rel_namespace(relid)));
-	cagg = ts_continuous_agg_find_by_view_name(NameStr(view_schema), NameStr(view_name));
+	view_name = get_rel_name(relid);
+	view_schema = get_namespace_name(get_rel_namespace(relid));
+	cagg = ts_continuous_agg_find_by_view_name(view_schema, view_name, ContinuousAggAnyView);
 
 	if (cagg == NULL)
 		return DDL_CONTINUE;
 
-	vtyp = ts_continuous_agg_view_type(&cagg->data, NameStr(view_schema), NameStr(view_name));
+	vtyp = ts_continuous_agg_view_type(&cagg->data, view_schema, view_name);
```

Comment:
Codecov complains that this line is not tested, which I really doubt.

Reply:
Yes, that does seem strange. But if this case isn't hit it isn't something introduced by the changes in this PR.

Reply:
Seems to be gone now.